Makefile

```diff
@@ -14,16 +14,12 @@ run-nohup:
 sync:
 	Rscript scripts/sync.R
 
-download:
+pull:
```

Comment:
these are basically synonyms, I added multiple so I didn't have to remember which one is actually defined